### ISSUE TITLE
How do people still restore their accounts for 12 13 years?

### ISSUE THREAD

**Issue by croatek** [Ezzpify/FindLowDigAccounts#12]
How do people still restore their accounts for 12 13 years?

**Comment by croatek** [comment]
Say that there is a bypass of technical support

**Comment by croatek** [comment]
Who knows the truth?

**Comment by ak0ya** [comment]
I did it today and got two accounts. talk to me on my skype: pandalishus

**Comment by failurexd9193** [comment]
@ak0ya can you please explain how you got passed the part where they require the CD Key? I have been trying to get an account and they wont let me login into it, or change the password.

**Comment by puuska** [comment]
we have our metods 😸

**Comment by hldsarchive** [comment]
yeah it may sound rude but we dont want to share new methods so valve could easily fix them
/id/-cx drop a  comment and invite i can help you guys but dont share new methods thanks

**Comment by puuska** [comment]
added //dodałem xD

**Comment by arlisk** [comment]
@hldsarchive  you can help me?

**Comment by troninoff** [comment]
@hldsarchive hi ?can u help me in 2025?
